source/wave_energy.rst

```diff
@@ -73,19 +70,19 @@ Potential Wave Power Resource Assessment
 Wave power per unit width of wave crest length transmitted by irregular waves can be approximated as
 
 .. math:: P_n = {{\rho * g}\over 16} H^2_s C_g (T_e,h)
-   :label: eq1
+   :label: wave_power
 
 where, :math:`P_n` is wave power (kW/m), :math:`\rho` is sea water density (1,028 kg m\ :sup:`-3`), :math:`g` is gravitational acceleration (9.8 m s\ :sup:`-2`), :math:`H_s` is significant wave height (m), and :math:`C_g` is wave group velocity (m s\ :sup:`-1`) as a function of wave energy period, :math:`T_e` (sec), and water depth :math:`h` (m) (Cornett 2008). :math:`C_g` can be estimated as
 
 .. math:: C_g = { {\left(1 + {{2kh}\over \sinh(2kh)}\right) \sqrt{{g\over k} \tanh(kh)}}\over 2 }
-   :label: eq2
+   :label: wave_group_velocity
 
 where the wave number :math:`k` is calculated using a dispersion relationship expressed as a function of wave frequency (:math:`w = 2\pi / T_e`) and water depth :math:`h`:
 
 .. math:: w^2 = {gk * \tanh(kh)}
-   :label: eq3
```

Comment:
Again wondering if these are referenced in the source code.

Reply:
Same as above, this label shouldn't affect how it's labeled in the built docs.